### ISSUE TITLE
Adding more functions for mode 1? 

### ISSUE THREAD

**Issue by Brink2Three** [boldstelvis/BMC_SBUS#3]
I'm attempting to add two more buttons to the first mode. I only need to change Focus, record, and ISO remotely so instead of doing a mode switch I was going to only use the first mode and add two buttons to that mode. I started trying to edit the example file myself by adding "pinRIGHT and pinLEFT" when defining IO, setting status and debounce, and setting pin state.

After doing this and testing it with a BMMCC, I was unable to get the two new buttons do send data. I'm guessing this is because you're only emulating one switch with one set of buttons, but I don't know how to change this to two switches by four buttons. 

I'll attach my .ino file, hopefully someone can explain what I'm doing wrong. 
Thanks! 

[BMC_SBUS_4Button Rev1.ino.zip](https://github.com/boldstelvis/BMC_SBUS/files/3439472/BMC_SBUS_4Button.Rev1.ino.zip)

**Comment by Brink2Three** [comment]
TL;DR, I defined the buttons, but didn't make them do things. Now they do things on their own channel. Code below.

I solved my own problem. I'll document it here for anyone that has the same issue.
I'll say first off, I'm not a programmer, coder or whatever you want to call it. My skill is in PC hardware, 3D modeling, and IT. So this was new to me. 

SO, in the file listed above, I have the new buttons defined, (pinRIGHT, pinLEFT, statusLEFT, statusRIGHT, etc) but I wasn't doing anything with them, and I hadn't assigned them to an sBus channel. It took me about two hours of reading the original document, the BMC_SBUS.h and BMC_SBUS.cpp files, plus the Futuba SBUS github and the "Futuba SBUS reverse engineered" posts such as [this](https://forum.arduino.cc/index.php?topic=99708.75) and [this](https://forum.arduino.cc/index.php?topic=99708.0) . After some time reading through those (and some trial and error if I'm honest.) I put more attention into the "  // ****SWITCH CHANNEL INPUT****" Segment of the code. I realized my mistake. 

After repeating that exact code and changing the references to earlier definitions (pin, status and chk variables) AND THE MODE + number segment (forgot about this at first and had 4 buttons toggling the same switch), my friend who was acting as my rubber ducky came in clutch and said "Don't you have to assign it to a different SBUS channel too?" and I figured it out. 

My current controller layout now has 4 buttons (Seen as two toggle switches on the sBUS end) and 1 10k Pot. I'm not using the mode switch as I can manually assign what channels I want to do what functions, and only need focus, Record switch, and shutter adjustments remotely. I'm attaching my updated file below. 
[BMC_SBUS_4BUTTON.ino.zip](https://github.com/boldstelvis/BMC_SBUS/files/3442827/BMC_SBUS_4BUTTON.ino.zip)
